skimage/transform/_geometric.py

```diff
@@ -1293,7 +1293,7 @@ class SimilarityTransform(EuclideanTransform):
 
     where ``s`` is a scale factor and the homogeneous transformation matrix is::
 
-        [[a0  b0  a1]
+        [[a0  -b0  a1]
```

Comment:
```suggestion
        [[a0  b0  a1]
```

Reply:
in my understanding, the equations are correct and the matrix is not.
the homogenous matrix should be:
[[s\*cos_theta, -s\*sin_theta, t_y],
[s\*sin_theta, s\*cos_theta, t_x],
[0,0,1]]
here:
a0 = s\*cos_theta
b0 = s\*sin_theta
a1 = t_x
b1 = t_y

in the correction you proposed there is a contradiction, as b0=s\*sin_theta=-s\*sin_theta